### PR TITLE
[CMS-631] [CMS-623] Improve advise command output and account for in-progress conversion.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

File: src/Commands/AdviseCommand.php [hunk 1]
```diff
@@ -62,33 +65,105 @@ public function advise(string $siteId, array $options = ['skip-upgrade-checks' =
...
+     */
+    private function adviseDevAlreadyOnDrupalRecommended(): void
+    {
+        $this->output()->writeln(
+            <<<EOD
+Advice: We recommend that this site be converted to use "drupal-recommended" Pantheon upstream:
+Drupal Recommended (drupal-recommended)
+This process has already been started and seems to be ready in the dev environment. To finish it, you should change the upstream with the following command:
+    {$this->getTerminusExecutable()} site:upstream:set {$this->site()->getName()} drupal-recommended
+You may run the conversion:advise command again to check your progress and see the next steps again.
```
Comment (greg-1-anderson) [inline src/Commands/AdviseCommand.php:138]: This is the last step; once the upstream is set, then they're done. Perhaps this message should say that they can run the advise command again to confirm that the conversion was completed successfully.

File: src/Commands/AdviseCommand.php [hunk 2]
```diff
@@ -62,33 +65,105 @@ public function advise(string $siteId, array $options = ['skip-upgrade-checks' =
...
+
+    /**
+     * Print advise for dev environment already on Drupal Recommended.
+     */
+    private function adviseDevAlreadyOnDrupalRecommended(): void
+    {
+        $this->output()->writeln(
+            <<<EOD
+Advice: We recommend that this site be converted to use "drupal-recommended" Pantheon upstream:
+Drupal Recommended (drupal-recommended)
```
Comment (greg-1-anderson) [inline src/Commands/AdviseCommand.php:135]: Add a little whitespace & indentation around this text, here and elsewhere, as shown in the example in the story (CMS-631)

**Comment by greg-1-anderson** [comment]
Looks good, just have a couple minor requests above. Also, could you show an example of the output from running the advise command in each of the scenarios, either here or on the JIRA card?

**Review by greg-1-anderson — CHANGES_REQUESTED** [review]
See above

**Comment by kporras07** [comment]
@greg-1-anderson done.

Added output examples to CMS-623

File: src/Commands/Traits/ConversionCommandsTrait.php [hunk 3]
```diff
@@ -525,4 +525,13 @@ protected function getTerminusExecutable(): string
...
         return $this->terminusExecutable = 'terminus';
     }
+
+    /**
+     * Determines whether a multidev named "conversion" exists for this site.
+     */
+    protected function conversionMultidevExist(): bool
+    {
+        $environments = $this->site()->getEnvironments()->fetch()->ids();
+        return in_array('conversion', $environments);
```
Comment (jabxjab) [inline src/Commands/Traits/ConversionCommandsTrait.php:535]: tip: strict comparison is preferable for `in_array` (i.e. `in_array('conversion', $environments, true);`)

File: src/Commands/Traits/ConversionCommandsTrait.php [hunk 4]
```diff
@@ -525,4 +525,13 @@ protected function getTerminusExecutable(): string
 
         return $this->terminusExecutable = 'terminus';
     }
+
+    /**
+     * Determines whether a multidev named "conversion" exists for this site.
+     */
+    protected function conversionMultidevExist(): bool
```
Comment (jabxjab) [inline src/Commands/Traits/ConversionCommandsTrait.php:532]: tip: `isConversionMultidevExist`

**Review by greg-1-anderson — APPROVED** [review]
LGTM!

**Pull request merged**